Source/GitReleaseManager/Configuration/ConfigurationProvider.cs

```diff
@@ -75,5 +79,18 @@ private static string GetConfigFilePath(string targetDirectory)
         {
             return Path.Combine(targetDirectory, "GitReleaseManager.yaml");
         }
+
+        private static void EnsureDefaultConfig(Config configuration)

```

Comment:
Wouldn't this duplicate the initial setting of the configuration here: https://github.com/GitTools/GitReleaseManager/pull/138/files#diff-2a4e85d209b40d54dc265c4582680d32R23-R25 ?

Reply:
No, I don't think it does.  During my testing, there were two code paths.  The first is in the case of there being no yaml file in the directory.  In this case a new instance of the `Config` class is created, using all the default values specified in the lines that you link to.  However, when you do have a yaml file, it only deserializes the contents of the file, i.e. it doesn't use the default values specified in the constructor.  As a result, I created this method to ensure that the default values are always set.

Reply:
I see. Got it